server/common.ts

```diff
@@ -428,6 +429,28 @@ export function isAdmin(request: express.Request, response: express.Response, ne
 	}
 }
 
+export async function hasProperExportKey(request: express.Request, response: express.Response, next: express.NextFunction) {
+	let exportKey = await getSetting<string>("exportKey");
+
+	if (exportKey === "") {
+		// No export key defined
+		response.status(403).json({
+			"error": "Fetching exports by key is not supported - please configure it in registration's admin menu"
+		});
+	} else {
+		let userKey = request.params.exportKey;
+		console.log("Yasss", userKey, exportKey);
```

Comment:
Can you please remove this `console.log`?

Reply:
awkward